src/lib/optimizer/join_ordering/enumerate_ccp.cpp

```diff
@@ -126,7 +126,7 @@ void EnumerateCcp::_enumerate_cmp(const JoinGraphVertexSet& primary_vertex_set)
     reverse_vertex_indices.emplace_back(current_vertex_idx);
   } while ((current_vertex_idx = neighborhood.find_next(current_vertex_idx)) != JoinGraphVertexSet::npos);
 
-  for (auto iter = reverse_vertex_indices.rbegin(); iter != reverse_vertex_indices.rend(); ++iter) {
+  for (auto iter = reverse_vertex_indices.rbegin(); iter != reverse_vertex_indices.rend(); ++iter) {  // NOLINT
```

Comment:
What does the linter complain about here?

Reply:
Changed to `  // NOLINTNEXTLINE(modernize-loop-convert)`.

Reply:
Reverse range views can be quite nice (https://stackoverflow.com/a/66053335), but our experience with range view and old GCC versions are not good.

Reply:
(just checked, we would need to require GCC >=10 for ranges)